docs/docs/build/bob-sdk/gateway.md

```diff
@@ -140,15 +140,7 @@ async function createTxWithOpReturn(
     opReturn
   );
   const psbt = unsignedTx.toPSBT(0);
-  const psbtHex = hex.encode(psbt);
-  const signedPsbtHex = psbtHex;
-  const signedTx = SigTx.fromPSBT(
-    bitcoin.Psbt.fromHex(signedPsbtHex).toBuffer()
-  );
-  signedTx.finalize();
-  const tx = bitcoin.Transaction.fromBuffer(Buffer.from(signedTx.extract()));
-
-  return tx;
+  return bitcoin.Transaction.fromPSBT(psbt);
```

Comment:
I think best to return this as a base64 encoded PSBT since this is what the wallets will expect for signing.

Reply:
This essentially

```ts
return Buffer.from(signedTx.extract());
```